feature-libs/asm/customer-360/components/sections/asm-customer-section/asm-customer-section.component.html

```diff
@@ -0,0 +1,7 @@
+<ng-container
+  [cxComponentWrapper]="{
+    flexType: component,
+    typeCode: component,
+    uid: ''
+  }"
```

Comment:
[MAJOR QUESTION]
Can you help me understand, why we use ComponentWrapperDirective for
rendering and passing data to specific Customer 360 section components?
Why we load those components as CMS components? 

1. What are the benefits of using CMS components for this purpose? 

2. What downsides of this approach that you can see? Does using CMS components for this purpose cause a UI delay, because of a need to call for OCC CMS for this component's data? Btw. what are the data that we actually need from CMS for those components? I could see that the layout and implementation of those components is driven only by the static Spartacus config. I have problem to understand what an added value it gives us to leverage the mechanism of CMS components here.

Did you consider any other alternative approaches that doesn't use CMS components?

Reply:
As discussed offline, CMS components were not necessary. They are removed in separate PR https://github.com/SAP/spartacus/pull/17874